src/zmq/zmqnotificationinterface.cpp

```diff
@@ -29,6 +29,14 @@ CZMQNotificationInterface::~CZMQNotificationInterface()
     }
 }
 
+std::list<const CZMQAbstractNotifier*> CZMQNotificationInterface::GetActiveNotifiers() const
+{
+    std::list<const CZMQAbstractNotifier*> result;
+    for (const auto* n : notifiers)
```

Comment:
Missing `{}`.

Reply:
Done.  I've seen single-line blocks without it (even in this file), but the style guide is indeed pretty clear that this should be avoided *going forward*.